File: yeongseong/2week/피자 나눠 먹기.js
function solution(n) {
    for (let i = 1; i < n; i++){
        // 변수 i를 시작하여 n 보다 작을 때까지 1씩 증가시킵니다.
        if(6 * i % n === 0){
            // 6 * i를 n으로 나눈 나머지가 0인 경우를 찾습니다.            
            return i;
        }
    }
}

File: gang/week2/피자 나눠 먹기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        for(int i = 6; ; i+=6) {    //피자 한판이 6조각이므로 6조각씩 증가
            if(i%n ==0) {   // 피자 조각을 인원으로 나누어 맞아 떨어지면
                answer = i / 6; //피자 판의 갯수를 구하는 것이기 때문에 나누기 6으로 판 개수를 구함
                break;
            }
        }
        return answer;
    }
}